src/test/java/UItest/Frontend_Test.java

```diff
@@ -0,0 +1,57 @@
+package UItest;
+
+import org.junit.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.web.server.LocalServerPort;
+import org.testcontainers.junit.jupiter.Testcontainers;
+import static org.junit.Assert.assertTrue;
+
+@Testcontainers
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+public class Frontend_Test {
+
+    @LocalServerPort
+    private int port;
+
+    /**
+     * User go to the homepage, click on the "Create Patient" button and create an association
+     * then submit the patient. After that, user should be able to see created event list in event list.
+     * @throws InterruptedException
+     */
+    @Test
+    public void UserCreatePatientAndSubmitPatient() throws InterruptedException {
+        //Go to the homepage
+        WebDriver driver = new ChromeDriver();
```

Comment:
is there a way to create browser-independent drivers? It would be nice to know our app works correctly for all browsers.

Reply:
this is a nice test! :)